src/main/java/io/sniffy/Spy.java

```diff
@@ -37,6 +36,28 @@ protected void addExecutedStatement(StatementMetaData statementMetaData) {
         executedStatements.add(statementMetaData);
     }
 
+    private volatile ConcurrentHashMap<String, AtomicLong> socketOperations = new ConcurrentHashMap<String, AtomicLong>();
+    private final String UNKNOWN = "unknown";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/sniffy/sniffy/file/1633591162/issues/source?bid=3005960&fileBranchId=3034483#l40)